activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb

```diff
@@ -391,6 +391,25 @@ def default_index_type?(index) # :nodoc:
 
       private
 
+        def add_index_opclass(column_names, options = {})
+          opclass = case options[:opclass]
+                    when String
```

Comment:
I think this misses Symbol?

Seems like it might read more easily if it were flipped a bit, making this the `else` clause -- then there's no need for a separate empty-hash branch.